WoWsShipBuilder.ViewModels/ShipVm/UpgradePanelViewModelBase.cs

```diff
@@ -20,7 +20,7 @@ public UpgradePanelViewModelBase(Ship ship, Dictionary<string, Modernization> up
             List<Modernization> filteredModernizations = upgradeData.Select(entry => entry.Value)
```

Comment:
Remove unnecessary conditional access

Reply:
Why? that conditional access is not equal to "all nations". it's correct for it to be like that

Reply:
Wrong wording, should have been "null-safe access". The question marks are no longer necessary, both for old and new code here.